Add image exif data

howdu opened the pull request.

awcodes commented:
This all looks good to me. But I'm concerned about how it will affect people who may have already run the upgrade command.

Seems like those users would need to manually add the exif column to their database.

Thoughts?

howdu commented:
Cheers please see recent commit.

I've added a `glob` for migrations beginning with 'upgrade_*.php'. Also added a hasColumn check to safely run upgrade command multiple times.

awcodes commented:
Looks good to me. thanks.
